src/h5web/visualizations/containers/hooks.ts

```diff
@@ -17,3 +24,27 @@ export function useDatasetValues(
 
   return valuesReader();
 }
+
+export function useChildDatasetValues(
```

Comment:
You could argue that this utility function is specific to NeXus and should live in `nexus/utils`.